pytorch_widedeep/datasets/_base.py

```diff
@@ -0,0 +1,47 @@
+from importlib import resources
+import pandas as pd
+
+
+def load_bio_kdd04(as_frame: bool = False):
```

Comment:
In my opinion all datasets should be in the same format (`csv`?) 

Not a major issue, but...

Reply:
We can ignore for now